### ISSUE TITLE
TypeError: Cannot read property 'dataFrame' of undefined after update to 6.6.0

### ISSUE THREAD

**Issue by ankon** [grafana/grafana#21817]
**What happened**:

One of the dashboard panels stopped working, and this error is logged in the browser console:
~~~~
react-dom.production.min.js:196 TypeError: Cannot read property 'dataFrame' of undefined
    at t.renderValue (BarGaugePanelEditor.tsx:60)
    at GraphSeriesToggler.tsx:29
    at Array.map (<anonymous>)
    at t.render (GraphSeriesToggler.tsx:28)
    at Za (react-dom.production.min.js:181)
    at $a (react-dom.production.min.js:180)
    at Tu (react-dom.production.min.js:261)
    at Ts (react-dom.production.min.js:230)
    at Os (react-dom.production.min.js:229)
    at ms (react-dom.production.min.js:223)
~~~~

**What you expected to happen**:

Panel should render the results of this query as bar chart:
```
sum(min_over_time(service_kafka_offset{kubernetes_namespace=~"$Environment"}[15m]) - max_over_time(service_s3_offset{kubernetes_namespace=~"$Environment"}[15m]) > 0) by (kubernetes_namespace,tenant,topic)
```

The results of the query should typically be empty, i.e. I expect a "No data" response.

**How to reproduce it (as minimally and precisely as possible)**:

Unknown

**Anything else we need to know?**:

I tried to change the time frame to include a time where there was data to exclude a "grafana doesn't handle the empty response", but even with data in the  `/api/datasources/proxy/2/api/v1/query_range` response the error is displayed.

**Environment**:
- Grafana version: Grafana v6.6.0 (5c11bbdfb4)
- Data source type & version: Prometheus
- OS Grafana is installed on: AWS Linux 2/EKS host for the default grafana 6.6.0 docker imager
- User OS & Browser: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4033.2 Safari/537.36
- Grafana plugins: grafana-clock-panel,grafana-simple-json-datasource,grafana-piechart-panel,mtanda-google-calendar-datasource,petrslavotinek-carpetplot-panel,natel-discrete-panel,jdbranham-diagram-panel,neocat-cal-heatmap-panel,snuids-radar-panel
- Others:

**Comment by marefr** [comment]
Duplicate of #21766

**Comment by marefr** [comment]
Fix coming in v6.6.1
